### DISCUSSION TITLE
I am getting errors while trying to use the V2 Twitter API

### DISCUSSION THREAD

**Discussion by ghost** [tweepy/tweepy#1957] [Questions]
I am getting the error " AttributeError: module 'tweepy' has no attribute 'Client' ". I have followed some forums saying to update tweepy and I have but there is no working outcome.

```
client = tweepy.Client(
    consumer_key="",
    consumer_secret="",
    access_token="",
    access_token_secret=""
)

client.status_update("test")
```

**Comment by Harmon758** [comment]
Make sure you don't have a file or folder in your path or current working directory named "tweepy" or "tweepy.py", so that you import the Tweepy library rather than your file or folder.

**Reply by ghost** [reply]
I don't.

**Reply by Harmon758** [reply]
What version of Tweepy are you using?

**Reply by fadlurss** [reply]
Finally, i have deleted file tweepy.py, then running others file, it's works! Thank you mate
